plugins/alias-finder/alias-finder.plugin.zsh

```diff
@@ -0,0 +1,46 @@
+alias-finder() {
+  local cmd=""
+  local exact=""
+  local longer=""
+  for i in $@; do
+    case $i in
+      -e|--exact) local exact=true;;
```

Comment:
You don't need to redefine a variable as local, put all your variable definitions at the top and then don't use `local` again. You can also define multiple variables in the same line:
```zsh
local cmd="" exact="" longer=""
```